pkg/descheduler/node/node.go

```diff
@@ -24,6 +24,7 @@ import (
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
+	kerr "k8s.io/apimachinery/pkg/util/errors"
```

Comment:
nit: for consistency can we name it `utilerrors`?
```suggestion
	utilerrors "k8s.io/apimachinery/pkg/util/errors"
```

Reply:
done